### PR TITLE
ci: use new orb packing to deploy orb (IN-303)

### PR DESCRIPTION
**Pull request by Xavientois**
**Fixes or implements IN-303**

### Brief description. What is this change?

This extracts all of the executors, jobs, and commands from the massive orb file and breaks it up into separate files to allow for orb packing

### Implementation details. How do you make this change?

I wrote a CLI tool to automate the process after I realized how slow it was to do it manually. @xavidop we might consider making my CLI tool open-source for anyone else who wants to perform a similar migration. I didn't find any similar tools online.

### Checklist

- [ ] this is a breaking change and should publish a new major version
- [ ] appropriate tests have been written

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
+    steps:
+      - checkout
+      - run:
+          name: Copy configuration from subdirectories
+          command: |
+            for DIR in commands jobs executors examples; do
+              # Copy all files in all subdirectories into the root
+              find src/$DIR -type f -exec mv {} src/$DIR \; || :
+            done;
+            pwd
```
Comment (xavidop) [inline .circleci/config.yml:52]: remove these debug commands

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
\ No newline at end of file
+          pub-type: production
+
+
+jobs:
+  # Copies configs from their respective subdirectories into
+  # the appropriate root directories
+  # Necessary because of https://github.com/CircleCI-Public/circleci-cli/issues/755
+  collect-section-files:
+    docker:
```
Comment (xavidop) [inline .circleci/config.yml:41]: use one of our executors

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,45 +1,57 @@
...
+      - run:
+          name: Copy configuration from subdirectories
+          command: |
+            for DIR in commands jobs executors examples; do
+              # Copy all files in all subdirectories into the root
+              find src/$DIR -type f -exec mv {} src/$DIR \; || :
+            done;
+            pwd
+            ls src
+      - persist_to_workspace:
```
Comment (xavidop) [inline .circleci/config.yml:54]: is this needed?

**Review by xavidop — CHANGES_REQUESTED** [review]
great job!!! I love the new structure! is super clean!

File: .circleci/config.yml (hunk 2 above)
Comment (Xavientois) [reply .circleci/config.yml:41]: Do we have an executor that is more lightweight than busybox?

**Review by xavidop — APPROVED** [review]
WAY TO GO!

**Pull request merged**